stdlib/public/Concurrency/Clock.cpp

```diff
@@ -83,6 +83,29 @@ void swift_get_time(
       *seconds = suspending.tv_sec;
       *nanoseconds = suspending.tv_nsec;
       return;
+    case swift_clock_id_wall:
+      struct timespec wall;
+#if defined(__linux__) || defined(__APPLE__) || defined(__wasi__) || defined(__OpenBSD__) || defined(__FreeBSD__)
+      clock_gettime(CLOCK_REALTIME, &wall);
```

Comment:
`timespec_get(&wall, TIME_UTC)` is the C standard API for this and is supported on Windows too.